ocfweb/docs/docs/staff/backend/prometheus.md

```diff
@@ -0,0 +1,44 @@
+[[!meta title="Prometheus"]]
+
+We use Prometheus to provide real-time monitoring of our hardware. The master is [[dementors|doc staff/backend/servers]] which
```

Comment:
s/hardware/servers

Reply:
well, we monitor desktops as well.

Reply:
And technically printers. Maybe `infrastructure`?

Reply:
Desktops and printers are also listed in [servers](https://www.ocf.berkeley.edu/docs/staff/backend/servers/) so I'll go ahead and link that under hardware.